CodeEntropy/main_mcc.py

```diff
@@ -5,6 +5,7 @@
 
 # import numpy as np
 import pandas as pd
+import yaml
```

Comment:
Is this a dep that needs installing in the pyproject.toml? So did you have to pip install pyyaml?